### PR TITLE
HW01 Completed

### PR DESCRIPTION
**Pull request by brendac29**
Name: Brenda Castaneda

I edited the README to add a brief introduction of myself to the class. I included four markdown text elements (headers, italics, link, list). I used the [Writing Documents with R Markdown](https://monashbioinformaticsplatform.github.io/2017-11-16-open-science-training/topics/rmarkdown.html) for tips and shortcuts on writing in markdown with special elements. I followed the homework workflow video step by step to get the README on RStudio and then push it back to the repository. I encountered a minor issue when I attempted to push the file to Github because R would say the process was complete but the changes would not appear on the repository. I figured out I had to mark the checkbox on the README file rather than just select it on R. Otherwise, the process was very simple.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # hw01
 
-## Brief Biography
-My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Carribean Studies*. 
\ No newline at end of file
+### About Me
```
Comment (Yucheng98) [inline README.md:3]: Nice headers and emphasis using both bold and italic. Very clear.

Comment (Yucheng98) [inline README.md:3]: Nice headers and emphasis using both bold and italic. Very clear.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Caribbean Studies*. Some of my policy interest areas are education, immigration, and foreign policy. 
+
+### Interest in this course
+I am taking this course to satisfy the Methods requirement for the Public policy major. I took [Social Science Inquiry](http://collegecatalog.uchicago.edu/search/?P=SOSC%2013100-13200-13300) last year and was introduced to social science research tools and data analysis using R. Computing for the social sciences was recommended to me by a friend and I am looking forward to brushing up on and expanding upon what I learned in SSI last year.
```
Comment (Yucheng98) [inline README.md:7]: The link is well imbedded, and it was glad to know about the inquiry.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+
+### Interest in this course
+I am taking this course to satisfy the Methods requirement for the Public policy major. I took [Social Science Inquiry](http://collegecatalog.uchicago.edu/search/?P=SOSC%2013100-13200-13300) last year and was introduced to social science research tools and data analysis using R. Computing for the social sciences was recommended to me by a friend and I am looking forward to brushing up on and expanding upon what I learned in SSI last year.  
+
+### Quaratine Activities
+I am currently in Chicago but spent my summer in quarantine at home in Dallas. Here's a list of some of the things I did to make life a little less boring:
+
+* taking long walks by the lake shore
+* baking
+* painting/DIY projects
```
Comment (Yucheng98) [inline README.md:14]: Bullet points are well wrote, very neat.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
-My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Carribean Studies*. 
\ No newline at end of file
+### About Me
+My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Caribbean Studies*. Some of my policy interest areas are education, immigration, and foreign policy. 
+
+### Interest in this course
+I am taking this course to satisfy the Methods requirement for the Public policy major. I took [Social Science Inquiry](http://collegecatalog.uchicago.edu/search/?P=SOSC%2013100-13200-13300) last year and was introduced to social science research tools and data analysis using R. Computing for the social sciences was recommended to me by a friend and I am looking forward to brushing up on and expanding upon what I learned in SSI last year.  
+
+### Quaratine Activities
+I am currently in Chicago but spent my summer in quarantine at home in Dallas. Here's a list of some of the things I did to make life a little less boring:
```
Comment (nbrown20) [inline README.md:10]: I like the bullet points here. I also painted during quarantine!

File: README.md [hunk 5]
```diff
@@ -1,4 +1,14 @@
 # hw01
 
-## Brief Biography
-My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Carribean Studies*. 
\ No newline at end of file
+### About Me
+My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Caribbean Studies*. Some of my policy interest areas are education, immigration, and foreign policy.
```
Comment (nbrown20) [inline README.md:4]: Good use of italics.

File: README.md (hunk 3 above)
Comment (nbrown20) [inline README.md:14]: Overall, nice job! It could have been nice to have a picture, but obviously that wasn't required!

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |            x  |            |
| **Presentation**            |    x      |              |            |
| **Achievement, creativity** |           |    x          |            |
| **Reproducibility**         |           |      x        |            |

Very clean appearance. Everything is rendered correctly and uses relevant formatting

**Pull request closed without merging**